lib/assets/javascripts/twine.js.coffee

```diff
@@ -283,6 +283,15 @@ Twine.bindingTypes =
         $(node).attr(key, value || null)
       lastValue = newValue
 
+  'bind-prop': (node, context, definition) ->
+    fn = wrapFunctionString(definition, '$context,$root', node)
+    lastValue = {}
+    return refresh: ->
+      newValue = fn.call(node, context, rootContext)
+      for key, value of newValue when lastValue[key] != value
+        $(node).prop(key, value || null)
```

Comment:
So maybe this should be `value ? null` so if it's evaluated to undefined it will instead set it to null, but other values that may be relevant (0, false) will be preserved